CompuCell3D/CMakeLists.txt

```diff
@@ -29,10 +29,10 @@ if(${CMAKE_SYSTEM_NAME} STREQUAL Windows)
     FIND_PACKAGE(PythonLibs)
     FIND_PACKAGE(PythonInterp)
     # INCLUDE(FindPythonLibs)
-    set(Python_EXECUTABLE ${PYTHON_EXECUTABLE} CACHE PATH "Python executable" FORCE)

```

Comment:
Edits to build scripts may cause conflicts with other current PRs. Are they necessary here?

Reply:
This is what I am fighting all the time on my windows machine. If we keep FORCE argument I cannot configure Python in CMAKE GUI - it defaults to some weird paths. If I remove FORCE I can specify Python paths with no problem. Maybe you have a good solution for me @tjsego  so that I can keep FORCE arg in and get correct Python?

Reply:
resolved the issue with addition new CMAKE option